python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -277,8 +279,25 @@ def groupby_agg(
     return new_dd_object(graph, gb_agg_name, _meta, divisions)
 
 
+def _redirect_aggs(arg):
+    """ Redirect aggregations to their corresponding name in cuDF
+    """
+    redirects = {sum: "sum", max: "max", min: "min"}
```

Comment:
```suggestion
    redirects = {sum: "sum", max: "max", min: "min", list: "collect", "list": "collect"}
```

Once #8045 is merged, this could also be added to match Dask.dataframe's behavior for collect list aggregations.

Reply:
Should we wait for #8045 before merging?

Reply:
Don't think so, as this is a small change and could just be added to #8045 if this is merged first. Either way is good for me